pyproject.toml

```diff
@@ -1,67 +1,49 @@
-[project]
-name = "llm_vm"
+[tool.poetry]
+name = "llm-vm"
 version = "0.1.0"
 description = "Taming LLMs to do whatever you want"
-authors = [
-    {name = "Anarchy Team", email = "matt@anarchy.ai"},
-]
-
-requires-python = ">=3.10" # this could be lower if need be
+authors = ["Anarchy Team <matt@anarchy.ai>"]
+license = "MIT"
 readme = "README.md"
-license = {text = "MIT"}
-dependencies = [
-    "dynaconf>=3.2.0",
-    "openai>=0.27.6",
-    "numpy>=1.22.4",
-    "watchdog>=3.0.0",
-    "xdg>=6.0.0",
-    "accelerate>=0.21.0",
-    "transformers>=4.31.0",
-    "flask>=2.3.2",
-    "flask_cors>=4.0.0",
-    "llama-index>=0.7.7",
-    "python-Levenshtein>=0.21.1",
-    "bs4>=0.0.1",
-    "sentencepiece>=0.1.99",
-    "sentence-transformers>=2.2.2",
-    "requests>=2.26.0",
-    "spacy>=3.5.3",
-    "outlines>=0.0.1",
-    "en_core_web_md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.6.0/en_core_web_md-3.6.0-py3-none-any.whl",
-    "gradio>=3.42.0",  # Specify the latest version of gradio
-]
 
-[project.optional-dependencies]
-dev = [
-    "black>=23.7.0",
-    "mypy>=1.5.1",
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
+dynaconf = ">=3.2.0"
+openai = ">=0.27.6"
+numpy = ">=1.22.4"
+watchdog = ">=3.0.0"
+xdg = ">=6.0.0"
+accelerate = ">=0.21.0"
+transformers = ">=4.31.0"
+flask = ">=2.3.2"
+flask_cors = ">=4.0.0"
+llama-index = ">=0.7.7"
+python-Levenshtein = ">=0.21.1"
+bs4 = ">=0.0.1"
+sentencepiece = ">=0.1.99"
+sentence-transformers = ">=2.2.2"
+requests = ">=2.26.0"
+spacy = ">=3.5.3"
+outlines = ">=0.0.1"
+en_core_web_md = {url = "https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.6.0/en_core_web_md-3.6.0-py3-none-any.whl"}
```

Comment:
```
spacy = ">=3.5.3"
```
but `en_core_web_md` is `3.6.0`, it may create some divergence, we should fix them, and as I said in my other comment, exact matching will be better for all dependencies too!